question28/js/password.js

```diff
@@ -1,8 +1,15 @@
 import { validateInputValue } from "./modules/validations";
+import { togglePassword } from "./modules/togglePassword";
 
 const submitButton = document.getElementById('js-submit-button');
 const passwordInputArea = document.querySelector(".js-password");
 const confirmPasswordInputArea = document.querySelector(".js-confirm-password");
+const passwordToggleButtons = document.querySelectorAll('.js-toggle-password-button');
+
+const url = new URL(window.location.href);
+const params = url.searchParams;
+
+if(params.get('token') !== localStorage.getItem('tokenForforgotPassword')) window.location.href = ('../notautherize.html');
```

Comment:
If the token value in the parameter and the token value in the local storage are incorrect, the user will be redirected to '/notautherize.html'.